src/environments/environment.development.ts

```diff
@@ -0,0 +1,4 @@
+export const environment = {
```

Comment:
What will `environment.development` be used for?

Reply:
Its used when I use ng serve on my local machine, tried deleting it but doesn't work locally